app/routes/settings.js

```diff
@@ -1,60 +1,24 @@
 const applications = require('../data/applications')
 const _ = require('lodash');
+const SettingsHelper = require('../data/helpers/settings')
 module.exports = router => {
 
   router.post('/admin/settings/new-cycle', (req, res) => {
 
     var settings = req.session.data.settings
 
+    // Turn it off
     if(settings.includes('new-cycle')) {
-      req.session.data.applications = applications.filter(app => {
-        if (app.status === 'Deferred' && app.cycle === '2019 to 2020') {
-          return false;
-        } else {
-          return true;
-        }
-      })
-      .filter(app => {
-        if (app.status === 'Conditions pending' && app.cycle === '2019 to 2020') {
-          return false;
-        } else {
-          return true;
-        }
-      })
+      req.session.data.applications = SettingsHelper.getMidCycleApplications(applications)
       _.remove(settings, function(item) { return item == 'new-cycle' })
-    } else {
-      let deferredPast = applications
-        .filter(app => app.cycle == "2019 to 2020")
-        .filter(app => (app.status == 'Deferred'))
-
-      let acceptedPast = applications
-        .filter(app => app.cycle == "2019 to 2020")
-        .filter(app => (app.status == 'Conditions pending'))
-
-      let other = applications
-        .filter(app => app.status !== 'Received')
-        .filter(app => app.status !== 'Interviewing')
-        .filter(app => app.status !== 'Deferred')
-        .filter(app => app.status !== 'Offered')
-        .filter(app => app.status !== 'Conditions pending')
-        .filter(app => app.status !== 'Recruited')
-
-      req.session.data.applications = deferredPast.concat(acceptedPast).concat(other);
 
+    // Turn it on
+    } else {
+      req.session.data.applications = SettingsHelper.getStartOfCycleApplications(applications)
       settings.push('new-cycle')
     }
-    res.redirect('/admin/settings')
+    res.redirect('/')
```

Comment:
Redirecting to the applications list rather than back to the same page doesn't make it obvious you've successfully changed the settings.